file/prefetch_test.cc

```diff
@@ -2353,314 +2353,6 @@ TEST_P(PrefetchTest1, SeekParallelizationTest) {
   Close();
 }
 
-// This test checks if readahead_size is trimmed when upper_bound is reached.
```

Comment:
Are these tests not valid anymore?

Reply:
No! These are for FindReadAheadSizeUpperBound that has been removed.

Reply:
I've additional tests for sync scan for block cache lookup in async support PR.